pyfolio/tears.py

```diff
@@ -1503,6 +1506,10 @@ def create_perf_attrib_tear_sheet(returns,
 
     return_fig : boolean, optional
         If True, returns the figure that was plotted on.
+
+    styles : list, optional
+        list of styles to show in separate plots. If None, style factors
+        and sectors will be shown in the same plot
```

Comment:
What would you think about making this more generic, and just supporting arbitrary partitions of the factors? Seems like this is less about these being style factors, and more about just splitting up the plots. Maybe something like a dict mapping the factor categories to the factor names?

So for what we're doing here, we could pass:
```
{'sector': ['sector_1', 'sector_2', ... ], 'style': ['style_1', 'style_2', ... ]}
```

Reply:
Yup I like that - I was hesitant to do it with the plots being side by side, but I'll put each plot on its own row as per @jzhang18's feedback, and then pass in a dict instead